app/src/editor/shared/components/modules/TextField.jsx

```diff
@@ -70,12 +70,11 @@ export default class TextFieldModule extends React.Component {
                         onMessage={this.onMessage}
                         onActiveChange={this.onActiveChange}
                     />
-                    <TextControl
-                        commitEmpty
+                    <Text
                         flushHistoryOnBlur
                         onCommit={this.onChange}
                         placeholder="Enter your text here"
-                        immediateCommit={false}
+                        smartCommit
```

Comment:
`immediateCommit` is now `smartCommit`. Why? `immediateCommit` is the default = as you type, each letter triggers `onCommit`. To commit on blur or `Enter` key you raise `smartCommit` flag.

`Text` is meant to be a bare (visually) `input` or `textarea`. Visuals are in `CoreText` and `MarketplaceText`.

FYI.